File: leetcode/344. Reverse String.cpp
class Solution {
public:
    void reverseString(vector<char>& s) {
        int begin = 0, end = s.size() -1;
        char tmp;
        while(begin < end){
            tmp = s[begin];
            s[begin++] = s[end];
            s[end--] = tmp;
        }        
    }
};

File: leetcode/344. Reverse String.py
class Solution:
    def reverseString(self, s: List[str]) -> None:
        """
        Do not return anything, modify s in-place instead.
        """
        length = len(s)
        count = 0
        while count< len(s):
            s.insert(length,s[0])
            s.remove(s[0])
            count +=1
            length -= 1
        